src/plugins/visualizations/public/vis_types/vis_type_alias_registry.ts

```diff
@@ -27,9 +34,49 @@ export interface VisualizationListItem {
   type?: BaseVisType | string;
 }
 
+export interface SerializableAttributes {
+  [key: string]: unknown;
```

Comment:
This is poorly defined. Is it possible to do better? At very least, a code comment should explain.